glue/core/link_manager.py

```diff
@@ -30,6 +30,7 @@
 from glue.core.subset import Subset
 from glue.utils import unbroadcast
 from glue.core.coordinate_helpers import dependent_axes
+from glue.plugins.join_on_key.link_helpers import Join_Link
```

Comment:
I think the class should probably just be called JoinLink (no underscore)

Reply:
Good point, I have renamed.